src/components/DateField/DateField.tsx

```diff
@@ -19,6 +20,13 @@ export interface DateFieldProps extends DateInputProps {
    */
   maxDate?: Date
 
+  /**
+   * Prop to show 'calendar Outline' as default date field icon.
+   * When this prop don't receive any value and icon is undefined,
+   * the date field will not shown with a icon.
+   */
+  hasDefaultIcon?: boolean
```

Comment:
Is this prop really necessary? You could just use `<DateField icon={null} />` to omit the icon?